src/test/a11y/tests/in-scotland.js

```diff
@@ -13,5 +13,5 @@ const iLiveInScotland = [
 ]
 
 module.exports = {
-  iLiveInScotland
+  iLiveInScotland: inScotland
```

Comment:
There is an issue with this export - it should be:
```
module.exports = {
  inScotland
}
```
This might give you some issues elsewhere.